src/OpenTK.GraphicsLibraryFramework/Enums/WindowAttributeGetter.cs

```diff
@@ -1,97 +1,96 @@
 //
-// WindowAttribute.cs
+// WindowAttributeSetter.cs
 //
 // Copyright (C) 2019 OpenTK
 //
 // This software may be modified and distributed under the terms
 // of the MIT license. See the LICENSE file for details.
 //
 
-// TODO: sort by data types?
 namespace OpenToolkit.GraphicsLibraryFramework
 {
     /// <summary>
-    /// Window related attributes.
+    /// Used to get window related attributes.
     /// </summary>
-    /// <seealso cref="IGLFW.WindowHint(WindowAttribute,bool)"/>
-    public enum WindowAttribute
+    /// <seealso cref="IGLFW.GetWindowAttrib"/>
+    public enum WindowAttributeGetter
```

Comment:
I recognized that we unfortunately need something like that and (WindowAttributeSetter respectively). Because WindowSetAttribute can not set everything which WindowGetAttribute can get. And these are only a subset of WindowHintBool as well. therefore I created two seperate enums for these, which reference WindowHintBool enum values.
I think it does make sense to do it like that. But I'm always open for better solutions(or better names^^)

Reply:
That makes sense, at least for now. If I get think of anything better though, I'll let you know.